examples/image_segmenter/README.md

```diff
@@ -18,90 +18,4 @@ wget https://download.pytorch.org/models/fcn_resnet101_coco-7ecb50ca.pth
     curl http://127.0.0.1:8080/predictions/fcn -T examples/image_segmenter/fcn/persons.jpg
     ```
 * Output
-
-```json
-[[[11.49452    11.49452    11.49452    ... 10.846567   10.846567
```

Comment:
Might be worth keeping the output of example inputs somewhere for people to cross verify. Maybe ``example_output.json``

Reply:
I can update this example. As far as I could tell, image_segmenter never actually worked correctly. So, I had to delete this big mass of text to replace it with a working example.